gent_base.theme

```diff
@@ -716,3 +716,37 @@ function gent_base_preprocess_pager(&$variables) {
   $variables['items']['first']['text'] = '1';
   $variables['items']['last']['text'] = $total;
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_node__news__teaser(&$variables) {
+  _gent_base_generate_placeholder_image($variables);
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_node__event__teaser(&$variables) {
+  _gent_base_generate_placeholder_image($variables);
+}
+
+/**
+ * Helper function to get generate a placeholder teaser.
+ *
+ * This requires the system_stream_wrapper module!
```

Comment:
Best in je helper functie een check inbouwen of deze module actief staat of niet. Zo crashen andere installaties niet bij het toevoegen van deze helper functie.
Nog beter is om de module als dependency op te geven in je theme. Maar dat is technisch nog niet mogelijk denk ik.

Reply:
Inderdaad de dependency is niet mogelijk momenteel. Dus heb ik een check ingebouwd.